docs/en/observability/landing-page/data/recommended-data.json

```diff
@@ -0,0 +1,141 @@
+{
+  "title": "What do you want to observe?",
+  "tabIcons": true,
+  "thumbnailImages": true,
+  "tabs": [
+    {
+      "title": "APM",
+      "id": "apm",
+      "icon": "apmApp",
+      "items": [
+        {
+          "title": "Monitor your application performance",
+          "description": "Learn how to collect and visualize Application Performance Monitoring (APM) data with the Elastic Stack.",
+          "link": "ingest-traces.html",
+          "image": "ingest-traces.png"
+        },
+        {
+          "title": "Monitor your application logs",
+          "description": "Correlate your application logs and traces for increased visibility into blah blah",
+          "link": "application-logs.html",
+          "image": "application-logs.png"
+        },
+        {
+          "title": "Integrate with OpenTelemetry",
+          "description": "Reuse your existing OTel instrumentation to easily send observability data to the Elastic Stack.",
+          "link": "{apm-guide-ref}/open-telemetry.html",
+          "book": "the APM guide",
+          "image": "open-telemetry.png"
+        }
+      ]
+    },
+    {
+      "title": "Infrastructure",
+      "id": "infrastructure-monitoring",
+      "icon": "metricsApp",
+      "items": [
+        {
+          "title": "Get started",
+          "description": "Learn how to monitor logs and infrastructure metrics from systems and services across your organization",
+          "link": "logs-metrics-get-started.html",
+          "image": "ingest-logs-metrics-uptime.png"
+        },
+        {
+          "title": "Integrations",
+          "description": "Stream in and visualize logs, metrics, traces, content, and more from all the sources in your ecosystem.",
+          "link": "https://docs.elastic.co/integrations",
+          "book": "the Integration docs",
+          "image": "integrations.png"
+        }
+      ]
+    },
+    {
+      "title": "RUM",
+      "id": "real-user-monitoring",
+      "icon": "rumApp",
+      "items": [
+        {
+          "title": "User experience",
+          "description": "Quantify and analyze the perceived performance of your web application.",
+          "link": "user-experience.html",
+          "image": "user-experience.png"
+        },
+        {
+          "title": "Monitor your users",
+          "description": "Deploy the Elastic APM Real User Monitoring (RUM) JavaScript Agent to add detailed performance metrics and error tracking of your web applications.",
+          "link": "{apm-rum-ref}/intro.html",
+          "book": "the RUM agent docs"
```

Comment:
@bmorelli25 do you have any ideas about the concept for a thumbnail illustration for this one? I'm out of ideas...

<img width="1166" alt="Screenshot 2023-05-09 at 5 21 52 PM" src="https://github.com/elastic/observability-docs/assets/10479155/a8da607c-595b-479c-8bf7-2feda4ec0856">

Reply:
What about something like this? This looks user related. I added it to the figma file if you like it.

<img width="223" alt="Screenshot 2023-05-09 at 5 28 11 PM" src="https://github.com/elastic/observability-docs/assets/5618806/c6d019d3-c54f-4d15-9406-f54fcb1c071a">